File: app/src/main/java/com/vladgba/keyb/SettingsFrag.java
package com.vladgba.keyb;

import android.os.Bundle;
import android.preference.PreferenceFragment;
import androidx.annotation.Nullable;

public class SettingsFrag extends PreferenceFragment {
    @Override
    public void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        addPreferencesFromResource(R.xml.settings);
    }
}

File: app/src/main/java/com/vladgba/keyb/SettingsFrag.kt
package com.vladgba.keyb

import android.os.Bundle
import android.preference.PreferenceFragment

class SettingsFrag : PreferenceFragment() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        addPreferencesFromResource(R.xml.settings)
    }
}